packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -46,6 +49,17 @@ const memoizedAllConvertChatMessage = memoizeFnAll(
   }
 );
 
+/* @conditional-compile-remove(file-sharing) */
+const extractAttachedFilesMetadata = (metadata: Record<string, string>): FileMetadata[] => {
+  const fileMetadata = metadata['fileSharingMetadata'];
+  if (!fileMetadata) return [];
+  try {
+    return JSON.parse(fileMetadata);
+  } catch (e) {
+    return [];
```

Comment:
can we console log the error here? I'm always scared of simply swallowing errors

Reply:
Oh yes.

Reply:
`console.exception` so we know what caused the 'Failed to parse }" error.